src/util/git.js

```diff
@@ -293,12 +295,15 @@ export default class Git {
           parser.on('finish', done);
 
           parser.on('entry', (header, stream, next) => {
-            let string = '';
+            const decoder = new StringDecoder('utf8');
+            let fileContent = '';
+
             stream.on('data', (buffer) => {
-              string += buffer.toString();
+              fileContent += decoder.write(buffer);
             });
             stream.on('end', () => {
-              update(string);
+              fileContent += decoder.end();
+              update(fileContent);
```

Comment:
Since it isn't really used anymore, maybe `fileContent` should be removed completely, using `update(decoded.end())` instead